generator/integration_tests/golden/tests/golden_kitchen_sink_client_test.cc

```diff
@@ -257,6 +288,62 @@ TEST(GoldenKitchenSinkClientTest, ListServiceAccountKeys) {
   EXPECT_STATUS_OK(response);
 }
 
+TEST(GoldenKitchenSinkClientTest, AsyncAppendRows) {
```

Comment:
Does this count as the "generator golden current options" test you asked for?  I was just going to start looking into it.

Reply:
Maybe?  It only tests that bidirectional streaming RPCs do the right thing.  One could imagine that this is done correctly for such RPCs, but incorrectly for regular RPCs, or for streaming reads, or streaming writes, or long running (I would say etc. but I think that is the full list).

Reply:
Ack.  I'll work on hitting a call of each variety.